.pipelines/prod-release-tag.yml

```diff
@@ -32,9 +31,29 @@ stages:
       billingE2EBranchName: $(billing-e2e-branch-name)
       imageTag: ${{ parameters.aroImageTag }}
       rpImageAcr: ${{ parameters.rpImageAcr }}
+- stage: Deploy_PilotSector
+  dependsOn: [Deploy_CanarySector]
+  displayName: 🚀 Deploy Pilot Sector
```

Comment:
nit: can we throw in 👨‍✈️ / 👩‍✈️ in addition to or in replace of the rocket?

Reply:
Hmm, the py linter doesn't like my emoji extension. I'll try another.